lib/utils/logger.js

```diff
@@ -106,3 +106,13 @@ Object.values(methods).forEach(method => {
 });
 
 module.exports = logger;
+
+module.exports.logVerboseMessage = (...args) => {
```

Comment:
@AntonSBilan could you add JSDoc here, please?

Reply:
Why do you need this? According to logger util, `info` messages will be displayed when `config.logLevel` is set to `verbose || debug`. This wrapper seems unnecessary. Please, just use `logger.info`.